docs/assets/scss/_code.scss

```
@@ -33,6 +33,15 @@
             padding: 1rem;
             border-radius: 10px;
         }
+
+        table td:nth-child(2) {
+            display: grid;
+            background-color: #fff;
+
+            pre {
+                border-radius: 0px 10px 10px 0px;
```

I think if you set all the corners to square it will address the issue of offset rounded corners in a widescreen window:
```suggestion
                border-radius: 0px 0px 0px 0px;
```

---

Got it, thanks for the review! I've updated the border-radius as you suggested.

Let me know if that looks good!